packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { IHbarLimitService } from './IHbarLimitService';
+import { HbarLimitPlanRepository } from '../../db/repositories/hbarLimiter/HbarLimitPlanRepository';
+import { EthAddressPlanRepository } from '../../db/repositories/hbarLimiter/EthAddressPlanRepository';
+import { IDetailedHbarSpendingPlan } from '../../db/types/hbarLimiter/hbarSpendingPlan';
+import { SubscriptionType } from '../../db/types/hbarLimiter/subscriptionType';
+import { Logger } from 'pino';
+
+export class HbarLimitService implements IHbarLimitService {
```

Comment:
What is the relationship between the `HbarLimitService` class and `HbarLimit`? Do they replace each other?

Reply:
`HbarLimit` will be deprecated and deleted after we start using `HbarLimitService`.